float64 true_positive
float64 true_negative
float64 false_positive
float64 false_negative